Altinn.Platform.Authentication.SystemIntegrationTests/Tests/SystemUserTests.cs

```diff
@@ -0,0 +1,183 @@
+using System.Net;
+using Altinn.Platform.Authentication.SystemIntegrationTests.Clients;
+using Altinn.Platform.Authentication.SystemIntegrationTests.Domain;
+using Altinn.Platform.Authentication.SystemIntegrationTests.Utils;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Altinn.Platform.Authentication.SystemIntegrationTests.Tests;
+
+/// <summary>
+/// Class containing system user tests
+/// </summary>
+[Trait("Category", "IntegrationTest")]
+public class SystemUserTests
+{
+    private readonly ITestOutputHelper _outputHelper;
+    private readonly SystemRegisterClient _systemRegisterClient;
+    private readonly PlatformAuthenticationClient _platformAuthentication;
+
+    /// <summary>
+    /// Testing System user endpoints
+    /// </summary>
+    /// 
+    public SystemUserTests(ITestOutputHelper outputHelper)
+    {
+        _outputHelper = outputHelper;
+        _platformAuthentication = new PlatformAuthenticationClient();
+        _systemRegisterClient = new SystemRegisterClient(_platformAuthentication);
+    }
+
+    private async Task<SystemRegisterState> CreateSystemRegisterUser()
+    {
+        // Prerequisite-step
+        var token = await _platformAuthentication.GetTokenForClient("SystemRegisterClient");
+
+        var teststate = new SystemRegisterState()
+            .WithVendor("312605031")
+            .WithResource(value: "kravogbetaling", id: "urn:altinn:resource")
+            .WithToken(token);
+
+        await _systemRegisterClient.PostSystem(teststate);
+
+        return teststate;
+    }
+
+    /// <summary>
+    /// Verify that system is created
+    /// </summary>
+    [Fact]
+    public async Task CreateSystemUser()
+    {
+        // Prerequisite
+        var teststate = await CreateSystemRegisterUser();
+
+        // Prepare
+        var requestBody = await Helper.ReadFile("Resources/Testdata/SystemUser/RequestSystemUser.json");
+
+        requestBody = requestBody
+            .Replace("{systemId}", $"{teststate.SystemId}")
+            .Replace("{randomIntegrationTitle}", $"{teststate.Name}");
+
+        var manager = new AltinnUser
+        {
+            userId = "20012772", partyId = "51670464", pid = "64837001585",
+            //scopes = "altinn:authentication/systemuser.request.read"
+        };
+        var token = await _platformAuthentication.GetPersonalAltinnToken(manager);
+
+        //Act
+        var respons =
+            await _platformAuthentication.PostAsync("authentication/api/v1/systemuser/50692553", requestBody,
+                token);
+
+        //Assert
+        Assert.Equal(HttpStatusCode.Created, respons.StatusCode);
+    }
+
+    /// <summary>
+    /// Test Get endpoint for System User
+    /// Github: #765
+    /// </summary>
+    [Fact]
+    public async Task GetCreatedSystemUser()
+    {
+        const string alternativeParty = "50891151";
```

Comment:
Det er ikke nødvendgivis slik at partyID er lik på tvers av miljø.  Er denne testen laget for et spesielt miljø?